tiup/tiup-command-uninstall.md

```diff
@@ -0,0 +1,36 @@
+---
+title: tiup uninstall
+---
+
+# tiup uninstall
+
+The `tiup uninstall` command is used to uninstall the installed components.
+
+## Syntax
+
+```sh
+tiup uninstall <component1>:<version> [component2...N] [flags]
+```
+
+- `<component1>`: the name of a component to uninstall.
+- `<version>`: the version to uninstall. If omitted, it uninstalls all installed versions of a component. For security reasons, the `--all` option must be added when `<version>` is omitted, indicating that all versions of a component need to be uninstalled.
+- `[component2...N]`: multiple components or versions to uninstall.
+
+## Options
+
+### --all
+
+- Uninstalls all installed versions of the specified components. You can use this option when `<version>` is omitted.
+- Data type: `BOOLEAN`
+- Default: `false`
+
+### --self
+
+- Uninstalls TiUP itself: delete all data downloaded from the mirror, but keep the data generated by TiUP and its components. The data is stored in the directory specified by the `TIUP_HOME` environment variable. If `TIUP_HOME` is not set, the default value is `~/.tiup/`.
```

Comment:
@lucklove Please help clarify what is "the data stored in the directory". Is it the data to be deleted or retained? Thanks~
```suggestion
- Uninstalls TiUP itself. When this option is used, all data downloaded from the mirror is deleted, but the data generated by TiUP and its components is retained. The data is stored in the directory specified by the `TIUP_HOME` environment variable. If `TIUP_HOME` is not set, the default value is `~/.tiup/`.
```